ecs-cli/modules/cli/compose/project/project_parseV3.go

```diff
@@ -270,8 +273,21 @@ func logWarningForDeployFields(d types.DeployConfig, serviceName string) {
 	}
 }
 
-func getEnvironment() map[string]string {
-	env := os.Environ()
+func getEnvironment(wrkDir string) map[string]string {
+	env, err := opts.ParseEnvFile(filepath.Join(wrkDir, dotEnvFileName))
```

Comment:
What happens if the .env file doesn't exist?

Reply:
oh, then a file-not-exist err will be returned by `ParseEnvFile` and ignored by ln 284

Reply:
Ah right, didn't see that. one way we've checked for this pre-emptively is using `os.Stat`, but this works fine too.